VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -86,58 +94,70 @@
                 </div>
                 <div class='icon-button-dark' id='toggle-hand-raised-img' *ngIf='!isPrivateConsultation && !isHost'
                     colour='grey' (click)='toggleHandRaised()'>
-                    <span class='far fa-hand-paper'>
+                    <span>
+                        <fa-icon [icon]="['far', 'hand-paper']"></fa-icon>
                         <span class='context-item'> {{(handRaised
                             ? 'private-consultation-room-controls.hand-raise-toggle-true'
                             : 'private-consultation-room-controls.hand-raise-toggle-false') | translate
-                            }}</span></span>
+                            }}</span>
+                        </span>
                 </div>
                 <div *ngIf='!isPrivateConsultation && isHost' id='leave-hearing'
                     class='icon-button-dark hide-for-medium' data-module='govuk-button'
                     (click)='displayLeaveHearingPopup = true'>
-                    <span class='fad fa-sign-out-alt fa-flip-horizontal'></span><span class='context-item'>{{
+                    <fa-icon icon='sign-out-alt' flip="horizontal"></fa-icon><span class='context-item'>{{
                         'private-consultation-room-controls.leave-hearing' | translate }}</span>
                 </div>
                 <div *ngIf='!isPrivateConsultation && isHost' id='pause-hearing'
                     class='icon-button-dark hide-for-medium' data-module='govuk-button' (click)='pause()'
                     [hidden]='!canCloseOrPauseHearing()'>
-                    <span class='far fa-pause-circle'><span class='context-item'>{{
-                            'private-consultation-room-controls.pause' | translate }}</span></span>
+                    <span>
+                        <fa-icon [icon]="['far', 'pause-circle']"></fa-icon>
```

Comment:
Again this one has far but the others don't also this says it is part of the pro usage https://fontawesome.com/v5.15/how-to-use/on-the-web/referencing-icons/basic-use

Reply:
Some icons allow regular style as free use, if they matched and were available I prefered them.
https://fontawesome.com/v5.15/icons/pause-circle?style=regular